guides/v2.2/pattern-library/containers/slideouts-modals-overlays/slideouts-modals-overalys.md

```diff
@@ -133,12 +133,12 @@ _Example of a Confirmation Modal_
 
 The active modal should include the following elements:
 
-* Content container (Modal Window)
+* ccContent container (Modal Window)
```

Comment:
Verify that you meant to add "c" before almost every list item and "cc" before this one.

Also, this topic has a typo "overalys", but fixing that is out of scope.

Reply:
Yikes, thanks for catching that...